includes/Admin/Product_Categories.php

```diff
@@ -52,6 +52,16 @@ public function __construct() {
 	 */
 	public function enqueue_assets() {
 
+		if ( $this->is_categories_screen() ) {
+
+			wp_enqueue_script( 'wc-facebook-product-categories', facebook_for_woocommerce()->get_plugin_url() . '/assets/js/admin/product-categories.min.js', [
+					'jquery',
+					'wc-backbone-modal',
+					'jquery-blockui',
+					'jquery-tiptip',
+					'facebook-for-woocommerce-modal',
```

Comment:
```suggestion
				'jquery',
				'wc-backbone-modal',
				'jquery-blockui',
				'jquery-tiptip',
				'facebook-for-woocommerce-modal',
```
It looks like there are two tabs instead of one for indentation here.